source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/locomotion/velocity/config/anymal_d/agents/torchrl_ppo_cfg.py

```diff
@@ -0,0 +1,188 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import torch.nn as nn
+from dataclasses import MISSING
+
+from omni.isaac.lab.utils import configclass
+
+from omni.isaac.lab_tasks.utils.wrappers.torchrl.torchrl_ppo_runner_cfg import (
+    ClipPPOLossCfg,
+    CollectorCfg,
+    OnPolicyPPORunnerCfg,
+    ProbabilisticActorCfg,
+    ValueOperatorCfg,
+)
+
+
+class AnymalDActorNN(nn.Module):
```

Comment:
Question: How having the model manually defined in the agent config will support changing it from CLI (e.g.: using hydra)?

Reply:
I suppose we can add an argument to the torchrl CLI args to specify which model definition method the user would like to use if we want to support both methods?